esy.json

```diff
@@ -7,6 +7,7 @@
     "ocaml": " >= 4.2.0 < 4.10.0",
     "@opam/fix": "*",
     "@opam/ocamlfind": "*",
+    "@opam/js_of_ocaml": "*",
```

Comment:
I'd prefer that typical development builds not require a dependency on jsoo. Could we make a new `.json` file that extends this one, and adds jsoo? Or at the very least, make this a devDependency so that when we make releases it doesn't include jsoo prebuilt binaries needlessly.